corehq/apps/importer/templates/importer/excel_fields.html

```diff
@@ -107,6 +107,44 @@
             });
 
             $('#autofill').click(autofillProperties);
+
+            function validateSubmitButton() {
+                var invalidFields = $('.excel_field option:selected')
+                                      .filter(function(k, v) { 
+                                          return /\s+/.test(v.value.trim());
+                                      }).length;
+                if (invalidFields > 0) {
```

Comment:
could just be: `$('.form-actions [type=submit]').prop('disabled', invalidFields > 0)`